var/spack/repos/builtin/packages/arm-forge/package.py

```diff
@@ -130,6 +131,12 @@ class ArmForge(Package):
         description='Detect available PMU counters via "forge-probe" during install',
     )
 
+    variant(
+        "accept-eula",
+        default=False,
+        description='Accept the EULA',
```

Comment:
Black style check doesn't like single quotes:

```suggestion
        description="Accept the EULA",
```